.vsts-ci.yml

```diff
@@ -4,7 +4,7 @@ variables:
 resources:
   containers:
   - container: ubuntu1404
-    image: lightgbm/vsts-agent:ubuntu-14.04
+    image: lightgbm/vsts-agent:latest
```

Comment:
sorry,  I think the tag name should be changed to ubuntu-14.04.
I will fix it in docker hub

Reply:
Got it!

Reply:
@guolinke Done! Can we remove unused tags on Docker Hub?

Reply:
I mean `latest` and `test`.

Reply:
sure